keras/models.py

```diff
@@ -563,17 +552,14 @@ class accuracy in the logs to stdout at each epoch.
                 sample_weight_val = standardize_weights(y_val)
             val_ins = X_val + [y_val, sample_weight_val]
 
-        if show_accuracy:
-            f = self._train_with_acc
-            out_labels = ['loss', 'acc']
-        else:
-            f = self._train
-            out_labels = ['loss']
+        f = self._train
+        out_labels = ['loss'] + [metric.__name__ for metric in self.train_metrics]
 
         sample_weight = standardize_weights(y, class_weight=class_weight,
                                             sample_weight=sample_weight)
         ins = X + [y, sample_weight]
-        metrics = ['loss', 'acc', 'val_loss', 'val_acc']
+        metrics = ['loss'] + [metric.__name__ for metric in self.train_metrics] + \
```

Comment:
`\` is to be avoided. This line can be rewritten as several successive statements.